testsite/requirements.txt

```diff
@@ -4,6 +4,7 @@ Jinja2==3.1.1
 PyJWT==2.3.0
 python-dateutil==2.8.2
 monotonic==1.1
+cryptography>=2.2.2
```

Comment:
Use pinned versions (i.e. `==`) in testsite/requirements.txt and minium versions (i.e. `>=` in ) the top level (app/project) requirements.txt.

Reply:
It seems like this should be the other way around. Wouldn't you want to run your test on the latest available version, and pin your releases to the latest tested version?

That said, I'll go ahead and pin it to the latest version (37.x) since it seems to be working at the moment.

Reply:
We pin testsite/requirements.txt because the testsite is supposed to work out-of-the-box (ie. after a git clone). It is too easy for a dependency to break backward compatibility. Then it is an open issue here from someone that was interested to check it out.
The toplevel requirements.txt is using  `>=` versions because it is the one used in the package uploaded to pypi. Here we can't pin a version otherwise it becomes a nightmare downstream to install dependencies in a project using this app.

Reply:
Fair enough.